release-guidelines.md

```diff
@@ -106,12 +106,24 @@ Join our mailing lists and correspond with the developers directly. We also enco
 
 *   Developer List: dev@wso2.org
 *   Architecture List: architecture@wso2.org
-*   User Forum:[ StackOverflow](http://stackoverflow.com/questions/tagged/wso2is)
+*   User Forum: [StackOverflow](http://stackoverflow.com/questions/tagged/wso2is)
+
+## **Slack Channels**
+
```

Comment:
Let's add slack as just another point.

We need to change the header that have all these points, since it's listed under Mailing lists.

Reply:
Will be handled in a separate effort